src/components/text/Text.constants.ts

```diff
@@ -0,0 +1,32 @@
+import { TYPE_MARGINS, TYPE_COLORS, TTypeMargin, TTypeColor } from '../../foundations/type';
+
+import { TTextSize, TTextElement, TTextWeight } from './Text.types';
+
+const TEXT_SIZES: TTextSize[] = [25, 50, 75, 100, 200, 300];
+
+const DEFAULT_TEXT_SIZE: TTextSize = 100;
+
+const TEXT_ELEMENTS: TTextElement[] = ['p', 'span'];
+
+const DEFAULT_TEXT_ELEMENT: TTextElement = 'p';
+
+const TEXT_WEIGHTS: TTextWeight[] = ['regular', 'bold'];
+
+const DEFAULT_TEXT_WEIGHT: TTextWeight = 'regular';
+
+const DEFAULT_TEXT_MARGIN: TTypeMargin = 'both';
```

Comment:
If the value set here doesn't change from the `Type`(ography) component, should we just return `DEFAULT_TYPE_MARGIN as DEFAULT_TEXT_MARGIN` like the `Heading` component?

Maybe double check all these for both convenience components to make sure we are consistent?

Reply:
Done!